ens-improvement-proposals/ensip-xx-app-specific-records.md

```diff
@@ -0,0 +1,76 @@
+---
+description: App-specific records
+---
+
+# ENSIP-XX: App-specific records
+
+| **Author**    | Jeff Lau \<jeff@ens.domains> |
+| ------------- | ---------------------------------- |
+| **Status**    | Draft                              |
+| **Submitted** | 2023-08-28                         |
+
+### Abstract
+
+This ENSIP extends [ENSIP 9 (multichain address resolution)](ensip-9-multichain-address-resolution.md) and [ENSIP 11 (multichain address resolution)](ensip-11-multichain-address-resolution.md) and [ENSIP 5 (Text Records)](ensip-5-text-records.md) to include a way to app specific addresses and public keys.
+
+This allows a separation of concerns between a user's main Ethereum/EVM-chain account and a signing key specific to an application allowing easy rotation of those keys. 
+
+### Motivation
+
+Apps such as Farcaster that use Sign in with Ethereum as a primary login require the generation of a new EoA. It is also possible that you may want to split your address usage across different applications for address hygiene, security or privacy reason. Lastly with account abstraction becoming more popular, we may end up with an increasing amount of different addresses for different applications that man need to be connected.
```

Comment:
A technicality, but Farcaster wallet clients don't use the SIWE standard for account creation / usage.  A new account is created when you make a transaction to a contract, and then the address that made the transaction is now the owner of the account.